app/src/Modals/SumSubForm.js

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import PropTypes from "prop-types";
+import Logo from "assets/img/emote_trade_limit.svg";
+// import Button from "@material-ui/core/Button";
+import Link from "@material-ui/core/Link";
+
+import UpperBar from "./components/upperBar";
+import Header from "./components/header";
+
+import cn from "classnames/bind";
+
+import style from "./Modals.scss";
+
+const cx = cn.bind(style);
+
+const SumSubForm = props => {
+  const { closeModal } = props;
+
+  const routerLocation = useLocation();
+  const [isLoading, setLoading] = useState(true);
+
+  const initSumsub = useCallback((clientId, accessToken, externalUserId) => {
+    var id = idensic.init(
+      // selector of the WebSDK container (see above)
+      "#sumsub-container",
+      // configuration object (see settings in the demo)
+      {
+        // provide your clientId (can be seen in the demo)
+        clientId,
+        // access token for specific externalUserId
+        accessToken,
+        // may be some additional parameters, see the Demo to see which ones, e.g.
+        externalUserId
+      },
+      // function for the WebSDK callbacks
+      function(messageType, payload) {
+        // e.g. just logging the incoming messages
+        console.log("[SUMSUB DEMO] Sumsub message:", messageType, payload);
+      }
+    );
+  });
```

Comment:
Please add dependencies here, otherwise this will re-create the callback every render, by adding e.g. `[]` for no dependencies.

Reply:
Checked